src/app/features/city-staff-site-design/color-picker/color-picker.component.ts

```diff
@@ -22,6 +22,14 @@ export class ColorPickerComponent implements OnInit {
   protected hue!: string | null;
   @Input() color!: string | null | undefined;
   @Output() eventColor: EventEmitter<string | null> = new EventEmitter();
+
+  // Emit color from input
+  sendInputColor(event: Event | null) {
+    const target = event?.target as HTMLInputElement;
+    this.eventColor.emit(target.value);
+  }
```

Comment:
Very minor thing, but since Angular's binding model already has the value of the input bound to `color`, you do not need to actually inspect the event for the value. You could omit the parameter in the event handler altogether and simply do the following:
```typescript
sendInputColor() {
    this.eventColor.emit(this.color);
}
```

Reply:
There is also the issue that, if edited directly, the color might not be a valid hex color string, a pattern type validator can probably solve that.
As a future improvement, @lunox94 @elskwid, I'd recommend either moving the input field into `dp-color-palette` or pass a `FormControl` instance to it, so that validation and values can come from a single source.

Reply:
@drmollineda I'll make the change to use `this.color`. Good catch. I got pretty focused on making it work, not making it right. 😁 

I don't have a strong opinion about where this should live (here or down into the `dp-color-palette`) except to say that if felt lighter to add it at the level where the color was being displayed. In terms of validation, we could definitely improve it. For now, it feels good enough that the save action checks if this is a valid hex color and reports the error to the user. What do you think?

Reply:
I think that as long as we are aware of the caveats, this one is good enough for now. I expect users might use the palette selector more often than not, unless they have a specific color string they want to use, anyway.